pkg/kubelet/kubelet.go

```diff
@@ -2238,32 +2239,88 @@ func (kl *Kubelet) rejectPod(pod *api.Pod, reason, message string) {
 		Message: "Pod " + message})
 }
 
+func (kl *Kubelet) getNodeResources() api.ResourceList {
```

Comment:
Thanks for the comments. I think this is a good idea, but I want to focus on implementing #12744. Maybe we should open another PR to implement what you have suggested.